src/main/java/com/commercetools/sync/services/impl/CategoryServiceImpl.java

```diff
@@ -75,7 +77,9 @@ public CategoryServiceImpl(@Nonnull final CategorySyncOptions syncOptions) {
         return fetchMatchingResources(categoryKeys,
             () -> CategoryQuery
                 .of()
-                .plusPredicates(categoryQueryModel -> categoryQueryModel.key().isIn(categoryKeys)));
+                .plusPredicates(categoryQueryModel -> categoryQueryModel.key().isIn(categoryKeys))
+                .plusExpansionPaths(CategoryExpansionModel::parent));
```

Comment:
The expansion should effect performance, I assume we don't need to change CategoryService to use key instead of Id.